nemo_curator/scripts/prepare_fasttext_training_data.py

```diff
@@ -67,20 +67,19 @@ def attach_args(
         formatter_class=argparse.ArgumentDefaultsHelpFormatter,
     )
 ):
-    parser.add_argument(
-        "--input-data-dir",
-        type=str,
-        default=None,
-        help="Input directory consisting of .jsonl files that are accessible "
-        "to all nodes. Use this for a distributed file system",
+    argumentHelper = ArgumentHelper(parser)
+
+    argumentHelper.add_arg_input_data_dir()
+    argumentHelper.add_arg_input_local_data_dir()
```

Comment:
We should delete this, it's not actually used in the script and is a relic of an older version.